src/api/java/blusunrize/immersiveengineering/api/crafting/ClocheRenderFunction.java

```diff
@@ -31,6 +32,12 @@
 
 	Collection<Pair<BlockState, TransformationMatrix>> getBlocks(ItemStack stack, float growth);
 
+	// hook for dynamic cloche recipes
```

Comment:
I'd prefer a proper javadoc comment here, describing the context and parameters

Reply:
On it.